src/components/attendee/Input/Input.module.scss

```diff
@@ -0,0 +1,25 @@
+@use "scss/attendee/variables.scss"as *;
+@use "scss/attendee/space";
+@use "scss/attendee/border";
+@use "scss/attendee/theme";
+
+.input {
+  label::after {
+    content: attr(data-label);
+    position: absolute;
+    top: space.empty(0.75);
+    display: flex;
+    align-items: center;
+    padding: 0 0 0 space.empty(0.75);
+    height: 26px;
+    color: rgba(0, 0, 0, 0.6);
```

Comment:
We should avoid hard coded colors.

Reply:
I've made a `./src/scss/attendee/theme.scss` for this purpose, but also, would be nice to have these things named in Figma so we only add the most essential colors (trying to avoid https://en.wikipedia.org/wiki/God_object like with the previous `constants.scss` )

@alexey-petrov Can Design answer our questions about colors? If they have different opacity levels at different places it would make it hard to uniform colors.